dist/main.js

```diff
@@ -4657,8 +4657,13 @@ var require_lib3 = __commonJS({
     AbortError.prototype = Object.create(Error.prototype);
     AbortError.prototype.constructor = AbortError;
     AbortError.prototype.name = "AbortError";
+    var URL$1 = Url.URL || whatwgUrl.URL;
     var PassThrough$1 = Stream.PassThrough;
-    var resolve_url = Url.resolve;
+    var isDomainOrSubdomain = function isDomainOrSubdomain2(destination, original) {
+      const orig = new URL$1(original).hostname;
+      const dest = new URL$1(destination).hostname;
+      return orig === dest || orig[orig.length - dest.length - 1] === "." && orig.endsWith(dest);
```

Comment:
Is another way of writing this:
```suggestion
      return orig === dest || orig.endsWith(`.${dest}`);
```

Reply:
Ahh sorry, didn't realise it was codegen.

Reply:
No problemo 😄  - but you might have a nice PR to raise against `node-fetch` 😄.